src/main/kotlin/com/progreizh/animaliste/controllers/AccountController.kt

```diff
@@ -0,0 +1,84 @@
+package com.progreizh.animaliste.controllers
+
+import com.progreizh.animaliste.entities.Account
+import com.progreizh.animaliste.entities.AccountLoginForm
+import com.progreizh.animaliste.repositories.AccountRepository
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import java.util.Optional
+
+@RestController
+@RequestMapping("/accounts")
+class AccountController(private val repository: AccountRepository) {
+    /**
+     * Récupère tous les comptes.
+     */
+    @GetMapping
+    fun getAllAccounts(): ResponseEntity<List<Account>>{
```

Comment:
```suggestion
    fun getAccounts(): ResponseEntity<List<Account>>{
```
Je sais que dans le controller de l'animal les noms sont pareil mais je pense qu'il faut changer